docs/src/rules/id-match.md

```diff
@@ -42,17 +42,13 @@ function do_something() {
     // ...
 }
 
-obj.do_something = function() {
-    // ...
-};
-
```

Comment:
This is not flagged by the rule.

Reply:
This was mistakenly here, as the `properties` option is `false` by default. Perhaps we could move this into these examples:

https://deploy-preview-17633--docs-eslint.netlify.app/rules/id-match#properties